04-hosszukard.md

```diff
@@ -10,7 +10,7 @@
 
 4.  A markolatgombbal végrehajtott találatok csak akkor számítanak érvényesnek, ha az ellenfél maszkjának borítás nélküli rostélyát találják el. Csak olyan találatokért adható pont, ami a maszkon sérülést nem okoz, illetve javasolt kesztyűs kézzel takarni a gombot. A szükségtelen erővel végrehajtott markolatgomb akciók a szabálytalanságok második csoportjában foglaltak szerint büntetendő.
 
-5.  A találati felület nem-érvényes részére érkező találatok nem számítanak találatnak, nem állítják meg a menetet, és nem érvényteleníti az azt követő találatot.
+5.  A találati felület nem-érvényes részére érkező találatok nem számítanak találatnak, nem állítják meg a menetet, és nem érvényteleníti az azt követő találatot. Kivéve, ha a bíró úgy ítéli meg, hogy a helyzet veszélyes lehet bármelyik félre, vagy ellehetetleníti a további akciók megítélését.
```

Comment:
Ez máshol is le van írva, ez a rész arról szól, h hogyan lehet találatokat adni.